pkg/logging/journald_logger.go

```diff
@@ -200,3 +234,12 @@ func prepareJournalCtlDate(t string) (string, error) {
 	s := tm.Format("2006-01-02 15:04:05")
 	return s, nil
 }
+
+func GetJournaldImageNameField(image string) string {
+	imageName := image
+	if repo, tag := imgutil.ParseRepoTag(imageName); tag == "latest" {
```

Comment:
is it disturbing to to have a `latest` tag ?
 
I would rather check for empty tag using this [func](https://github.com/containerd/nerdctl/blob/main/pkg/imgutil/filtering.go#L352)

Reply:
Also, I'll add a test for the case where tag is latest.

Reply:
Thanks for the comments!!! @fahedouch 
When implementing tests, I noticed that Docker displays `the image name specified` when running a container in the IMAGE_NAME field ...

For example, we can check this behavior when running a container using the following image (`ghcr.io/containerd/busybox:latest`).

<details><summary>Details</summary>
<p>

```bash
> CONTAINER_NAME="latest-image"
> LATEST_IMAGE="ghcr.io/containerd/busybox:latest"

> d images | grep ghcr.io/containerd/busybox
ghcr.io/containerd/busybox         latest     16ea53ea7c65   3 years ago     1.24MB

> d run --rm --name $CONTAINER_NAME --log-driver=journald $LATEST_IMAGE sh -euxc "echo foo; echo bar"
+ echo foo
+ echo bar
foo
bar

> sudo journalctl -a CONTAINER_NAME=$CONTAINER_NAME -n 1 -o json-pretty | grep $LATEST_IMAGE
	"IMAGE_NAME" : "ghcr.io/containerd/busybox:latest",
```

</p>
</details> 

This behavior makes sense in the three cases above.
Therefore, I think it's not necessary to split the process according to the tag of image, so I'll fix the commit to delete `GetJournaldImageNameField()`.